### PR TITLE
Readme py2 req

### PR DESCRIPTION
**Pull request by jjzieve**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # GTC to VCF converter
 
+## Requirements
+
+**GTCtoVCF currently only works with python2** and the following packages are required.
```
Comment (mialam24) [inline README.md]: Do we know if it works with versions less than python 2.7, like python2.6? We could be specific and say something along the lines of "GTCtoVCF is currently only known to work with python2.7"

**Comment by mialam24** [comment]
Do we need the entire Requirements section at the top of the readme? It seems a bit "in your face". Maybe the disclaimer at the top that "GTCtoVCF is currently only known to work with python2.7" is suffice

**Comment by jjzieve** [comment]
> Do we need the entire Requirements section at the top of the readme? It seems a bit "in your face". Maybe the disclaimer at the top that "GTCtoVCF is currently only known to work with python2.7" is suffice

I sorta remember users having issues with wrong package versions used as well, not just the python version. The python version is a requirement, so it makes sense to me to keep it in that section instead of just being randomly on its own. I'd rather be "in your face" about everything necessary to run the script upfront than make it maybe more "friendly" by having the requirements at the bottom.

**Comment by mialam24** [comment]
> > Do we need the entire Requirements section at the top of the readme? It seems a bit "in your face". Maybe the disclaimer at the top that "GTCtoVCF is currently only known to work with python2.7" is suffice
> 
> I sorta remember users having issues with wrong package versions used as well, not just the python version. The python version is a requirement, so it makes sense to me to keep it in that section instead of just being randomly on its own. I'd rather be "in your face" about everything necessary to run the script upfront than make it maybe more "friendly" by having the requirements at the bottom.

Ok, sounds good to me

**Review by mialam24 — APPROVED** [review]

**Pull request merged**